config_test.go

```diff
@@ -30,6 +30,9 @@ func TestConfig(t *testing.T) {
 	if c.Incoming.Port != 1080 {
 		t.Error("incoming.port != 1080")
 	}
+	if c.Incoming.MetricsPort != 8081 {
+		t.Error("incoming.port != 1080")
```

Comment:
8081?
These days, I use https://github.com/stretchr/testify to write this kind of tests.

Reply:
I wrote it by copying and pasting, sorry :(
This is a simple struct comparison, and I believe google/go-cmp is more appropriate.

Reply:
I will fix this, soon.